capnslog/formatters.go

```diff
@@ -28,6 +29,28 @@ type Formatter interface {
 	Flush()
 }
 
+func NewSimpleFormatter(w io.Writer) *SimpleFormatter {
+	return &SimpleFormatter{
+		w: bufio.NewWriter(w),
+	}
+}
+
+type SimpleFormatter struct {
```

Comment:
Strangely enough, I was going to add some, but there weren't any others so I didn't. I'm happy to add them and rebase if you'd like, I'd just rather not add them for everything :)